Decred Wallet/Extensions/UIColor.swift

```diff
@@ -23,6 +23,9 @@ extension UIColor {
         static let offWhite = UIColor(hex:"#F3F5F6")
         static let lightOffWhite = UIColor(hex: "#F9FBFA")
         static let yellowWarning = UIColor.init(hex: "#FFC84E")
+        static let thinGray = UIColor.init(hex: "#a4abb1")
+        static let darkGray = UIColor.init(hex: "#4e5f70")
+        static let transparentThinGray = UIColor.init(hex: "#a4abb1", alpha: 0.3)
```

Comment:
I see you added the following new colors: 
`thinGray`
`transparentThinGray`

Why not reuse any of the following already defined colors?  `lightGray`, `lighterGray`, `offWhite` and `lightOffWhite`.

Reply:
There are  not new colors but form the ones used in storyboard.